### PR TITLE
Remove Variable 

### PR DESCRIPTION
**Pull request by jburel**
Remove variable no longer needed
This was required for Python 2.7 support on CentOS 6

### CODE REVIEW

**Comment by joshmoore** [comment]
Will a similar variable be needed for PY3? (Not that it needs adding now; I'm just wondering if this will need adding back in soon)

**Comment by jburel** [comment]
Maybe so I will revisit the cleaning and if it is not required
when we work on PY3  I can do a full cleaning

File: linux/step05_1_all_webapps.sh [hunk 1]
```diff
@@ -40,26 +40,10 @@ if [ $install_figure = true ]; then
 	rm $NAME_FIGURE_ZIP
 	mv figure* OMERO.server/lib/python/omeroweb/figure
 
-	echo "value=$PY_ENV"
 	# Install required packages
```
Comment (mtbc) [inline linux/step05_1_all_webapps.sh]: Is the comment still needed?

File: linux/step05_1_all_webapps.sh [hunk 2]
```diff
@@ -40,26 +40,10 @@ if [ $install_figure = true ]; then
...
-		set -u
-		pip install reportlab markdown
-	elif [ "$PY_ENV" = "py27_ius" ]; then
-		virtualenv -p /usr/bin/python2.7 /home/omero/omeroenv
-		set +u
-		source /home/omero/omeroenv/bin/activate
-		set -u
-		/home/omero/omeroenv/bin/pip2.7 install reportlab markdown
-		deactivate
 	else
```
Comment (mtbc) [inline linux/step05_1_all_webapps.sh]: Should this be `fi`?

File: linux/step05_2_websessionconfig.sh [hunk 3]
```diff
@@ -4,18 +4,7 @@ PY_ENV=${PY_ENV:-py27}
 
 cd ~omero
 
-echo "value=$PY_ENV"
 # Install required packages
```
Comment (mtbc) [inline linux/step05_2_websessionconfig.sh]: Is this comment still needed?

**Comment by jburel** [comment]
Thanks @mtbc 
It should be fixed now

**Comment by mtbc** [comment]
:+1: I don't see any other problems.

Outwith this PR, in the longer term we might want more `else`s that alert of unset/unexpected values.

**Comment by jburel** [comment]
This is mainly b/c that code is not really used. I might get rid of it since we install the apps differently now

**Pull request merged**